test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -174,6 +176,7 @@
       app
       [jetty9-service/jetty9-service
        webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
        status-service]
       (merge status-service-config
         {:web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/alternate-status"}})
```

Comment:
@cprice404 The extra testing that you were suggesting is covered by the new `status-debug-logging-test` now, right?